dcat/rdf/dcat.ttl

```diff
@@ -96,9 +96,9 @@ dct:PeriodOfTime
   skos:scopeNote "Začátek a konec intervalu může být dán vlastnostmi dcat:startDate či time:hasBeginning a dcat:endDate či time:hasEnd. Interval také může být otevřený, tj. může mít pouze začátek či pouze konec."@cs ;
 .
 dct:Location
-  skos:definition "A spatial region or named place."@en ;
-  skos:definition "A spatial region or named place."@es ;
-  skos:definition "Prostorová oblast či pojmenované místo."@cs ;
+  skos:scopeNote "A spatial region or named place."@en ;
```

Comment:
This wording is the same as official DCMI definition - http://dublincore.org/specifications/dublin-core/dcmi-terms/2012-06-14/?v=terms#Location